addon/utils/color.js

```diff
@@ -176,40 +176,42 @@ export function hsbToHex(color) {
  * => '282a36bf'
  */
 export function rgbaToHex(red, green, blue, alpha) {
-  const isPercent = `${ red }${ alpha || '' }`.toString().includes('%');
+  const isPercent = `${red}${alpha || ''}`.toString().includes('%');
 
   if (typeof red === 'string') {
     const res = red.match(/(0?\.?\d{1,3})%?\b/g).map(Number);
-    [ red, green, blue, alpha ] = res;
+    [red, green, blue, alpha] = res;
   } else if (typeof red === 'object') {
     ({ red, green, blue, alpha } = red);
   } else if (isPresent(alpha)) {
     alpha = parseFloat(alpha);
   }
 
-  if (typeof red !== 'number' ||
+  if (
+    typeof red !== 'number' ||
     typeof green !== 'number' ||
     typeof blue !== 'number' ||
     red > 255 ||
     green > 255 ||
-    blue > 255) {
+    blue > 255
+  ) {
     throw new TypeError('Expected three numbers below 256');
   }
 
   if (typeof alpha === 'number') {
     if (!isPercent && alpha >= 0 && alpha <= 1) {
       alpha = Math.round(255 * alpha);
     } else if (isPercent && alpha >= 0 && alpha <= 100) {
-      alpha = Math.round(255 * alpha / 100);
+      alpha = Math.round((255 * alpha) / 100);
     } else {
       throw new TypeError(`Expected alpha value (${alpha}) as a fraction or percentage`);
     }
-    alpha = (alpha | 1 << 8).toString(16).slice(1);
+    alpha = (alpha | (1 << 8)).toString(16).slice(1);
```

Comment:
Check these new parentheses don't break this calculation anywhere (+ same below)